src/pages/signup/terms.tsx

```diff
@@ -1,22 +1,32 @@
 import React from 'react';
 import { useState } from 'react';
```

Comment:
```suggestion
import { React, useState } from 'react';
```
이렇게 한줄로 줄일 수 있을 것 같아요!

Reply:
React는 default export에요!
```suggestion
import React, { useState } from 'react';
```